mlir/include/mlir/Dialect/Linalg/Passes.td

```diff
@@ -104,6 +104,11 @@ def LinalgGeneralizeNamedOpsPass : Pass<"linalg-generalize-named-ops"> {
   let dependentDialects = ["linalg::LinalgDialect"];
 }
 
+def LinalgSpecializeGenericOpsPass : Pass<"linalg-specialize-generic-ops"> {
```

Comment:
If this is already the case in this file, please keep things alphabetically sorted.

Reply:
It doesnt seem sorted. So kept it specialize nextto generalize.